code/modules/research/designs/autolathe_designs.dm

```diff
@@ -491,6 +491,14 @@
 	build_path = /obj/item/shovel
 	category = list("initial","Misc", "Tool Designs")
 
+/datum/design/pickaxe
+	name = "Pickaxe"
+	id = "pickaxe"
+	build_type = AUTOLATHE | PROTOLATHE
+	materials = list(/datum/material/iron = 100)
+	build_path = /obj/item/pickaxe
```

Comment:
If anything i would make this the mini pickaxe, as long as they have a worse mine speed. Auto lathe print a massive pickaxes seems goofy but its a nitpick

Reply:
> If anything i would make this the mini pickaxe, as long as they have a worse mine speed. Auto lathe print a massive pickaxes seems goofy but its a nitpick
i kinda agree but you can already autolathe shovels which are probably comparable in size

Reply:
Right now, mini pickaxe is the same speed, while being smaller. I guess I can make the improv pick smaller or something, I dunno.

Reply:
No wait. It's _already_ the size of a mini pick.